### ISSUE TITLE
Custom Overlays

### ISSUE THREAD

**Issue by eeadie** [Redth/ZXing.Net.Maui#142]
Hi,

Apologies if I have missed this in the documentation and it is already supported. I have previously used the Xamarin Forms version of this package which provided the ability to have a Custom Overlay during scanning. The default one consisted of a horizontal red line centred in the scanning window. Users of our app found this overlay useful when scanning barcodes.

Is there any plans to add support for Custom Overlays to ZXing.Net.Maui?

Many Thanks,

Euan

**Comment by juezen** [comment]
Same doubt (and need) here!

**Comment by jfversluis** [comment]
I don't think we have any immediate plans for this and should be easily achievable through what was shared above. Add it in the same grid and make sure the Z-index is correct to make any overlay you could possibly want!
